README.md

```diff
@@ -35,6 +35,10 @@
 
 ---
 
+> [!WARNING]
+> This is the working branch for the work in progress 2.0 SDK, see [this tracking issue](https://github.com/open-telemetry/opentelemetry-js/issues/5148) for details.
+> If you are a user looking for the currently released state, you are probably looking for the [1.x SDK](https://github.com/open-telemetry/opentelemetry-js/tree/v1.x) on the v1.x branch.
```

Comment:
```suggestion
> If you are a user looking for the current released state, you are probably looking for the [1.x SDK](https://github.com/open-telemetry/opentelemetry-js/tree/v1.x) on the v1.x branch.
```

Reply:
fixed in https://github.com/open-telemetry/opentelemetry-js/pull/5284/commits/2610521b1638265dd5f12bcdd72e5a36d19157c7 🙂